README.md

```diff
@@ -23,46 +23,46 @@
-- Web Axe: Color Contrast Resources - http://www.webaxe.org/color-contrast-tools/
+- [UMN: Accessibility & Web Design (Laura Carlson)](http://www.d.umn.edu/itss/training/online/webdesign/accessibility.html)
+- [UMN: Color](http://www.d.umn.edu/itss/training/online/webdesign/color.html)
+- [UMN: Interviews](http://www.d.umn.edu/itss/training/online/webdesign/misc.html)
+- [UMN: Usability](http://www.d.umn.edu/itss/training/online/webdesign/usability.html)
+- [UMN: Typography](http://www.d.umn.edu/itss/training/online/webdesign/type.html)
+- [UMN: Standards, Guidelines, & Patterns](http://www.d.umn.edu/itss/training/online/webdesign/standards.html)
+- [UMN: SVG & XML](http://www.d.umn.edu/itss/training/online/webdesign/xml.html#svg)
+- [Web Axe: Color Contrast Resources](http://www.webaxe.org/color-contrast-tools/)
 - Laura Kalbag: Animation https://laurakalbag.com/bookmarked-on-animation/
 
 ## Courses and Educational Resources
 - [Courses & Education](https://github.com/accessibility/a11y-courses)
 - [Podcasts, Videos & Twitch Streams (CivicActions)](https://accessibility.civicactions.com/guide/resources)
 
 ## Meet-up, Groups and Event Resources
-- A11Y Project Events - https://a11yproject.com/events.html
-- Accessibility and Inclusive Design (Deborah Edwards-Onoro) - https://www.lireo.com/accessibility-inclusive-design-in-person-groups/
-- UMN: Associations - http://www.d.umn.edu/itss/training/online/webdesign/associations.html
-- UMN: Events & Conferences - http://www.d.umn.edu/itss/training/online/webdesign/events.html
-- Web Axe: AccessMeetups - https://twitter.com/webaxe/lists/accessmeetups/members
-- Accessibility Meetup Survey - http://www.dennisdeacon.com/a11y-meetup-survey-2018/
-- A11yTalks - https://a11ytalks.com/
+- [A11Y Project Events](https://a11yproject.com/events.html)
+- [Accessibility and Inclusive Design (Deborah Edwards-Onoro)](https://www.lireo.com/accessibility-inclusive-design-in-person-groups/)
+- [UMN: Associations](http://www.d.umn.edu/itss/training/online/webdesign/associations.html)
+- [UMN: Events & Conferences](http://www.d.umn.edu/itss/training/online/webdesign/events.html)
+- [Web Axe: AccessMeetups](https://twitter.com/webaxe/lists/accessmeetups/members)
+- [Accessibility Meetup Survey](http://www.dennisdeacon.com/a11y-meetup-survey-2018/)
+- [A11yTalks](https://a11ytalks.com/)
 
 ## Government Resources
-- 18F's Tools List - https://accessibility.18f.gov/tools/
-- French Government Resources - Ressources RGAA - Collated by Access42 - http://gta2017.access42.net/2/ressources/
-- List Government Resources & Good Links from Lainey Feingold http://www.lflegal.com/2013/05/gaad-legal/
+- [18F's Tools List](https://accessibility.18f.gov/tools/)
+- [French Government Resources - Ressources RGAA - Collated by Access42](http://gta2017.access42.net/2/ressources/)
+- [List Government Resources & Good Links from Lainey Feingold](http://www.lflegal.com/2013/05/gaad-legal/)
```

Comment:
Looks like URL changed to https://www.lflegal.com/2013/05/gaad-2013/

Reply:
I wonder if the link should be this page: https://www.lflegal.com/resources/

Reply:
Thanks @weboverhauls. I think this list needs to be revised. A few other broken links so lets merge this and I'll new PR to fix them.